src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -45,20 +45,16 @@
 
         List<FeedbackSession> endEntities = load()
                 .filter("endTime >", curStart)
-                .filter("endTime <=", curEnd)
                 .list();
 
         List<FeedbackSession> startEntities = load()
-                .filter("startTime >=", curStart)
                 .filter("startTime <", curEnd)
                 .list();
 
         List<FeedbackSession> endTimeEntities = new ArrayList<>(endEntities);
         List<FeedbackSession> startTimeEntities = new ArrayList<>(startEntities);
 
-        endTimeEntities.removeAll(startTimeEntities);
-        startTimeEntities.removeAll(endTimeEntities);
-        endTimeEntities.addAll(startTimeEntities);
+        endTimeEntities.retainAll(startTimeEntities);
```

Comment:
`retainAll` will take quadratic time to run. We can use a `HashSet` to do this in linear time.

Reply:
Ya Hashset will be good i think

Reply:
I will try to implement that